provisioner/k8s/README.md

```diff
@@ -1,3 +1,112 @@
+A user can specify a particular `Bundle` to be available in the cluster for inspection before any application of the resources. 
+Given a `Bundle` resource named `my-bundle`, the k8s provisioner will pull down and unpack the bundle to a variety of storage backends, such as ConfigMaps or PersistentVolumes.
+
+Currently the ConfigMap backend is supported by the provisioner, and the content can be inspected by viewing the metadata ConfigMap, which holds
+information on all ConfigMaps related to this bundle. 
+
+`kubectl -n rukpak-system get cm --selector core.rukpak.io/owner-name=<bundle-name>` gets all ConfigMaps related to the `Bundle` on-cluster.
+
+Surfacing the content of a bundle in a more user-friendly way, via a plugin or additional API, is on the RukPak roadmap. 
+
+### Pivot between bundle versions 
+
+The `BundleInstance` API is meant to indicate the version of the bundle that should be active within the cluster. Given two unpacked bundles in the cluster,
+`my-bundle-v0.0.1` and `my-bundle-v0.0.2`, the `spec.bundleName` field of the related `BundleInstance` can be updated to pivot to a different version. For example by pivoting from an older version,
+to the newer version, by updating the `spec.bundleName` field the k8s provisioner will create the new bundle content on-cluster and remove the old content.  
+
+## Running locally 
+
+To experiment with the k8s provisioner locally, first setup a local cluster. 
+
+Once the cluster has been setup, take the following steps from the root of the repository:
+* Create the rukpak namespace via `kubectl create ns rukpak-system` by default
+* Run `make bin/k8s` to build the k8s provisioner binary locally
+* Apply the RukPak core CRDs via `make install-apis`
+* Run `./bin/k8s` to start the controller locally outside the cluster. 
+
+From there, creating some `Bundles` and `BundleInstance` types to see the provisioner in action. For an example bundle to use, 
+the [combo operator](https://github.com/operator-framework/combo) is a good candidate. 
+
+Create the bundle:
+```yaml
+---
+apiVersion: core.rukpak.io/v1alpha1
+kind: Bundle
+metadata:
+  name: combo-0.0.1
+spec:
+  image: quay.io/tyslaton/combo-plain-bundle:v0.0.1
+  provisionerClassName: core.rukpak.io/plain
+```
+
+Before creating the combo `BundleInstance` on-cluster, ensure that there is a namespace available that 
+supports the combo operator. 
+
+Create the namespace:
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: combo-system
+```
+
+Create the combo `BundleInstance` referencing the combo `Bundle` available in the cluster. 
+
+```yaml
+apiVersion: core.rukpak.io/v1alpha1
+kind: BundleInstance
+metadata:
+  name: combo-system
+spec:
+  provisionerClassName: core.rukpak.io/plain
+  bundleName: combo-0.0.1
+```
+
+From there, check out the combo operator and ensure that the CSV is present on the cluster. 
```

Comment:
nit: CSV is present on the cluster?

Reply:
whoops, that got through from the old doc